bin/kano-feedback-widget-cli

```diff
@@ -12,24 +12,36 @@
 """Kano Feedback Widget CLI
 
 Usage:
-  kano-feedback-widget-cli <qid> <response>
+  kano-feedback-widget-cli [--dry-run] [--debug] --question=<response>...
```

Comment:
I'm a bit worried here about the format of the responses because it will fail if there is a `:` in the response, instead you could do

```docopt
kano-feedback-widget-cli [--dry-run] [--debug] (question_id> <response>) ...
```

and then you could go to a list of tuples with

```python
zip(question_id, response)
```